tox.ini

```diff
@@ -9,25 +11,44 @@ envlist =
     coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
 extras = test
-deps = zope.testrunner
+
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+commands =
+    flake8 src setup.py
+    check-manifest
 
 [testenv:coverage]
-basepython =
-    python3
+basepython = python3
+deps =
+    coverage
+    zope.testrunner
 commands =
     coverage run -m zope.testrunner --test-path=src []
     coverage html
-    coverage report --fail-under=98
-deps =
-    zope.testrunner
-    coverage
+    coverage report -m
```

Comment:
Mildly worried that removal of the 98% threshold will now allow the coverage to backslide.

(Unless we have coveralls.io configured to track coverage and post commit statuses?)

Reply:
Yeah, https://coveralls.io/github/zopefoundation/z3 shows

> COVERAGE THRESHOLD FOR FAILURE  [  98.5%]
> COVERAGE DECREASE THRESHOLD FOR FAILURE [   0.2%]

which is, wow, I usually set the decrease threshold to 1%.